src/compiler/checker.ts

```diff
@@ -13327,6 +13380,24 @@ namespace ts {
             }
         }
 
+        function getLeftMostAliasDeclarationOfInternalModuleReference(node: EntityName): Declaration {

```

Comment:
Seems very similar to getLeftMostAliasDeclarationOfExpression. Is there a way to consolidate them?